### DISCUSSION TITLE
websocket error

### DISCUSSION THREAD

**Discussion by a3ro-dev** [EvieePy/Wavelink#316] [Q&A]
This is the old Lavalink websocket endpoint. Please use /v4/websocket instead. If you are using a client library, please update it to a Lavalink v4 compatible version or use Lavalink v3 instead.

**Comment by chillymosh** [comment]
@a3ro-dev did you mean to make this? If you did then you need to provide some actual info on what you are using.

I can only make the assumption you are using a pre 3.x version of Wavelink and can only recommend you update to the latest.

**Comment by TonnoBelloSnello** [comment]
@a3ro-dev Can you send me how you are trying to connect the socket? I had the same issue but i solved by changing the `uri` parameter in the Node connection:

Pre:
```python 
nodes = [wavelink.Node(uri="localhost:7000", password=os.environ.get("LAVALINK_PASSWORD"))]
```
post:
```python
nodes = [wavelink.Node(uri="http://localhost:7000", password=os.environ.get("LAVALINK_PASSWORD"))]
```

**Comment by a3ro-dev** [comment]
okay

**Reply by chillymosh** [reply]
@a3ro-dev In the future, please provide more details. Your post had no description of your setup other than a generic error message which can have multiple causes.
